azerothcore/azerothcore-wotlk
# refactor(creature-loot): improve chance % of Bohan NPC

Thanks to @BarbzYHOOL for this improvement.

## CHANGES PROPOSED:
-  improve the chance % to drop the main 2 items of this loot


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2740

## TESTS PERFORMED:
- I didn't test

## HOW TO TEST THE CHANGES:
- go to the NPC Boahn
```
.go creature id 3672
```
- kill the NPC and check the loot

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
